pipeline/frontend/yaml/constraint.go

```diff
@@ -37,13 +36,18 @@ type (
 
 // Match returns true if all constraints match the given input. If a single
 // constraint fails a false value is returned.
-func (c *Constraints) Match(metadata frontend.Metadata) bool {
+func (c *Constraints) Match(metadata frontend.Metadata, local bool) bool {
```

Comment:
maybe instead, add useragent to the metadata?
https://github.com/cncd/pipeline/blob/master/pipeline/frontend/metadata.go#L85